subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -258,9 +258,9 @@ Moreover, you may want to define your own shorthand tasks to fit your workflow.
 
 At times it is useful to generate the Ivy module descriptor file (normally `ivy.xml`) without publishing your module to an Ivy repository. Since descriptor file generation is performed by a separate task, this is very easy to do.
 
-The “`ivy-publish`” plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named “`generateDescriptorFileFor«_PUBNAME_»Publication`”, which will be “`generateDescriptorFileForIvyJavaPublication`” for the previous example of the “`ivyJava`” publication.
+The Ivy Publish Plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named `generateDescriptorFileFor«_PUBNAME_»Publication`, which will be `generateDescriptorFileForIvyJavaPublication` for the previous example of the `ivyJava` publication.
 
-You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to “`build/publications/«_PUBNAME_»/ivy.xml`”.
+You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to `build/publications/«_PUBNAME_»/ivy.xml`.
```

Comment:
See previous comment about "«_PUBNAME_»". Not sure I feel that strongly about it though.

Reply:
I've decided to leave these as-is for now.